Improved Buttons

Varniro opened the pull request:
Added styling for the buttons including hover shadow box effect and fixed padding.

I Would like to contribute fruther to the project to improve GUI.

ashawe reviewed (changes requested):
1) Add button:focus so that people can see if the button has been clicked. because it doesn't show the click animation. Maybe try reducing the shadow on focus so that it looks like being clicked.
2) Change cursor to pointer

Varniro commented:
- Added functionality to turn change the cursor to pointer when hovering above the buttons.
- Added visual effects to convey clicking effects better on the buttons to the user.
